Add Installation and Configuration for Benchmark section (#4124)
opensearch-project/documentation-website: _benchmark/index.md

---
layout: default
title: OpenSearch Benchmark
nav_order: 1
has_children: false
has_toc: false
---

# OpenSearch Benchmark

OpenSearch Benchmark is a macrobenchmark utility provided by the [OpenSearch Project](https://github.com/opensearch-project). You can use OpenSearch Benchmark to gather performance metrics from an OpenSearch cluster for a variety of purposes, including:

- Tracking the overall performance of an OpenSearch cluster.
- Informing decisions about when to upgrade your cluster to a new version.

> - Determining how changes to your workflow&#8212;such as modifying mappings or queries&#8212;might impact your OpenSearch cluster.

Reviewer: ```suggestion
- Determining how changes to your workflow&#8212;such as modifying mappings or queries&#8212;might impact your cluster.
```

OpenSearch Benchmark can be installed directly on a compatible host running Linux and macOS. You can also run OpenSearch Benchmark in a Docker container. See [Installing OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/installing-benchmark/) for more information.

> ## OpenSearch Benchmark documentation status

Reviewer: Is this section necessary? It essentially states the obvious, and I would prefer that we keep this kind of content out of the documentation.

Author: I'm with you. I'll remove it after I address the other comments.

> We are working diligently to provide complete documentation for OpenSearch Benchmark and will publish additional documentation as  To see the current status of documentation efforts, or to request a specific topic, please make an issue on the [OpenSearch documentation-website](https://github.com/opensearch-project/documentation-website) repo.

Reviewer 2: ```suggestion
We are working diligently to provide complete documentation for OpenSearch Benchmark and will continue to evolve the published documentation. To see the current status of documentation efforts or to request a specific topic, please create an issue on the [OpenSearch documentation-website](https://github.com/opensearch-project/documentation-website) repo.
```

> We are working diligently to provide complete documentation for OpenSearch Benchmark and will continue to evolve the published documentation. To see the current status of documentation efforts or to request a specific topic, please create an issue on the [OpenSearch documentation-website](https://github.com/opensearch-project/documentation-website) repo.

Reviewer: ```suggestion
We are working diligently to provide complete documentation for OpenSearch Benchmark and will continue to evolve the published documentation. To see the current status of documentation efforts or to request a specific topic, please create an issue in the [OpenSearch documentation-website](https://github.com/opensearch-project/documentation-website) repo.
```
